src/common/CMakeLists.txt

```
@@ -92,6 +92,7 @@ add_custom_command(OUTPUT scm_rev.cpp
       "${CMAKE_CURRENT_SOURCE_DIR}/scm_rev.h"
       # technically we should regenerate if the git version changed, but its not worth the effort imo
       "${CMAKE_SOURCE_DIR}/CMakeModules/GenerateSCMRev.cmake"
+      VERBATIM
```

```suggestion
    VERBATIM
```
Keeps tabbing consistent with the other parameters.

---

Addressed.